### DISCUSSION TITLE
Pause on lost focus

### DISCUSSION THREAD

**Discussion by bdotsamir** [ppy/osu#18512] [Ideas] [answered]
This would help tremendously- when I'm in the middle of a map (actually, more recently this happened when I was 98% finished but) and another program forcibly takes the focus away from osu!, the map will continue on like nothing happened, causing lost notes and such. Adding an option to enable pause on lost focus (or even making it the default) would be awesome :+1:

**Comment by frenzibyte** [comment, marked as answer]
Automatic pausing when losing focus should already be the case? Please state your platform and a video recording with FPS overlay open (via Ctrl+F11) as you switch focus to another window (while osu! is still visible after switch), to ensure it's operating properly.

**Reply by bdotsamir** [reply]
Sure, my platform is linux, specifically pop!_os 22.04. I'll attach a video in one moment.

**Reply by bdotsamir** [reply]
that's strange, it's not doing it now? hang on, the parameters are a little different now than when it was happening.

**Reply by bdotsamir** [reply]
im no longer able to reproduce this issue. weird. how can i close this discussion?
